cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -1250,6 +1251,15 @@ public void removeNotificationReceivedListener() {
     }
 
     public boolean fireNotificationOpenedListener(final NotificationOpenedResult openedResult) {
+        if (openedResult.getNotification().getAppBanner() != null) {
+            ActivityLifecycleListener.setActivityInitializedListener(new ActivityInitializedListener() {
```

Comment:
What if the Activity has already been initialized for some reason here? Then the listener will not fire?
Maybe we can make sure inside `ActivityLifecycleListener.setActivityInitializedListener` that if we already have the Activity, it will trigger the Listener right away?